app/models/bet_option.rb

```diff
@@ -4,4 +4,10 @@ class BetOption < ApplicationRecord
 
   scope :winners, -> { where(winner: true) }
   scope :losers, -> { where(winner: false) }
+
+  def winning_option_total
```

Comment:
I think this method should just be calculating the total bet. Not all bets are winners